### PR TITLE
partial logging: creep, worker, setup, mining

### PR DESCRIPTION
**Pull request by karlthepagan**
merge early, merge often :)

#563

### CODE REVIEW

File: creep.Setup.js [hunk 1]
```diff
@@ -61,24 +61,27 @@ let Setup = function(typeName){
         return memory;
     };
     this.isValidSetup = function(room){
-        if( room.controller.level < this.minControllerLevel ) {
+        let rcl = room.controller.level;
+        if( rcl < this.minControllerLevel ) {
             return false;
-            if (DEBUG && this.type === Creep.Setup.debugType) console.log(room.name, Creep.Setup.debugType, 'low RCL');
+            if (DEBUG && TRACE) trace('Setup', {setupType:this.type, room:room.name, rcl, Setup:'isValidSetup'}, 'low RCL');
```
Comment (cyberblast) [inline creep.Setup.js]: will never get called after return

Comment (karlthepagan) [inline creep.Setup.js]: good catch

Comment (karlthepagan) [inline creep.Setup.js]: Fixed!

File: creep.Setup.js [hunk 2]
```diff
@@ -61,24 +61,27 @@ let Setup = function(typeName){
         return memory;
     };
     this.isValidSetup = function(room){
-        if( room.controller.level < this.minControllerLevel ) {
+        let rcl = room.controller.level;
+        if( rcl < this.minControllerLevel ) {
+            if (DEBUG && TRACE) trace('Setup', {setupType:this.type, room:room.name, rcl, Setup:'isValidSetup'}, 'low RCL');
             return false;
```
Comment (karlthepagan) [inline creep.Setup.js]: Fixed!

**Pull request merged**